HdmiInput/HdmiInput.cpp

```diff
@@ -250,10 +255,25 @@ namespace WPEFramework
         uint32_t HdmiInput::readEDIDWrapper(const JsonObject& parameters, JsonObject& response)
         {
             LOGINFOMETHOD();
+            returnIfParamNotFound(parameters, "portId");
```

Comment:
What is portId? Is it same as deviceId in writeEDID? Can we then use the same names? Also we should retain existing behavior and return EDID of default port when portId is not specified.